### DISCUSSION TITLE
Librechat with Docker - frequently stops responding

### DISCUSSION THREAD

**Discussion by AumCoin** [danny-avila/LibreChat#4279] [Q&A]
The other day I noticed Librechat wasn't responding, even after a soft reboot.  I tried to do some diagnosis, failed, so I did:

```
docker compose down
docker compose up -d
```

which fixed it, even after a soft reboot.  However, today after a cold reboot it is down again.  Has anyone experienced this, or does anyone have any ideas what might be causing it or how to diagnose it?

**Comment by danny-avila** [comment]
No, this is not usual or expected. I have an instance that runs for months without restarting if I let it.

Need more information. What are the hardware specs of where you're running LibreChat on docker?

Also, try seeing if there's an error when it stops responding. If you are using docker, they are saved in the `./logs` directory at project root.

Check the latest error logs (files starting with "error" in name, i.e. `logs/error-2024-09-26.log`)

**Comment by AumCoin** [comment]
I checked the logs.  All logs are empty except a log from a couple days ago indicating a 404.  If I try to connect (and fail to connect) it doesn't generate a log entry.

I have 32GB of RAM with most of it free.  The relevant drive has 15GB available.  CPU is Celeron J4105.

**Comment by AumCoin** [comment]
The port seems to be listening but won't respond.  I should also add that there is no host firewall enabled.

```
root@br2:/var/log# sudo lsof -i -P -n | grep LISTEN
minidlnad  476 minidlna    8u  IPv4  13899      0t0  TCP *:8200 (LISTEN)
ollama     500   ollama    3u  IPv4    917      0t0  TCP 127.0.0.1:11434 (LISTEN)
sshd       529     root    3u  IPv4    895      0t0  TCP *:22 (LISTEN)
sshd       529     root    4u  IPv6    897      0t0  TCP *:22 (LISTEN)
apache2    585     root    3u  IPv6  13876      0t0  TCP *:80 (LISTEN)
apache2    587 www-data    3u  IPv6  13876      0t0  TCP *:80 (LISTEN)
apache2    588 www-data    3u  IPv6  13876      0t0  TCP *:80 (LISTEN)
smbd       697     root   46u  IPv6  14815      0t0  TCP *:445 (LISTEN)
smbd       697     root   47u  IPv6  14816      0t0  TCP *:139 (LISTEN)
smbd       697     root   48u  IPv4  14817      0t0  TCP *:445 (LISTEN)
smbd       697     root   49u  IPv4  14818      0t0  TCP *:139 (LISTEN)
docker-pr 1039     root    4u  IPv4  16025      0t0  TCP *:8080 (LISTEN)
docker-pr 1055     root    4u  IPv6  15172      0t0  TCP *:8080 (LISTEN)
docker-pr 1093     root    4u  IPv4  16683      0t0  TCP *:8082 (LISTEN)
docker-pr 1108     root    4u  IPv4  15229      0t0  TCP *:4750 (LISTEN)
docker-pr 1117     root    4u  IPv6  15232      0t0  TCP *:4750 (LISTEN)
docker-pr 1126     root    4u  IPv4  15235      0t0  TCP *:8081 (LISTEN)
docker-pr 1133     root    4u  IPv6  16087      0t0  TCP *:8081 (LISTEN)
docker-pr 1179     root    4u  IPv4  16102      0t0  TCP *:3080 (LISTEN)
docker-pr 1187     root    4u  IPv6  16105      0t0  TCP *:3080 (LISTEN)
docker-pr 1204     root    4u  IPv4  16727      0t0  TCP *:5333 (LISTEN)
docker-pr 1210     root    4u  IPv6  15276      0t0  TCP *:5333 (LISTEN)
docker-pr 1333     root    4u  IPv4  17463      0t0  TCP *:5800 (LISTEN)
docker-pr 1341     root    4u  IPv6  16765      0t0  TCP *:5800 (LISTEN)
root@br2:/var/log# ping br2
root@br2:/var/log# curl 127.0.0.1:3080
curl: (56) Recv failure: Connection reset by peer
```

**Reply by danny-avila** [reply]
Check all the container logs for all the containers associated with LibreChat, you should be able to see what's running with `docker ps` then run `docker logs <CONTAINER ID OR NAME> -n 100` (remove `<>`), which will print the last 100 lines, adjust as needed
